### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by vadym-valchak**
- [DEMO LINK](https://vadym-valchak.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://vadym-valchak.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job. A few changes required.

File: src/index.html [hunk 1]
```diff
@@ -15,8 +15,103 @@
...
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
```
Comment (volodymyr-soltys97) [inline src/index.html:32]:
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -15,8 +15,103 @@
...
+    <header class="header">
+      <div class="logo">
```
Comment (volodymyr-soltys97) [inline src/index.html:34]: This div is redundant
```suggestion
```

File: src/index.html [hunk 3]
```diff
@@ -15,8 +15,103 @@
...
+        <a
+          class="logo-link"
+          href="#"
+        >
+          <img
+            class="logo-img"
```
Comment (volodymyr-soltys97) [inline src/index.html:40]: Fix class 
```suggestion
          class="header__link"
          href="#"
        >
          <img
            class="header__logo"
```

File: src/index.html [hunk 4]
```diff
@@ -15,8 +15,103 @@
...
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="menu-items">
+          <li class="menu-item">
+            <a
+              href="#"
+              class="is-active menu-link"
```
Comment (volodymyr-soltys97) [inline src/index.html:51]: Fix class everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="is-active nav__link"
```

File: src/index.html [hunk 5]
```diff
@@ -15,8 +15,103 @@
...
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item">
```
Comment (volodymyr-soltys97) [inline src/index.html:56]: Add the blank line between the links
```suggestion
          </li>
          
          <li class="menu-item">
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,65 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+
+  --active-menu: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:10]: Create a variable with root selector

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,65 @@
...
+.menu-item:last-child {
+  margin-right: 0;
+}
+
+.menu-link {
+  display: block;
+  position: relative;
+  align-items: center;
+  font-weight: 500;
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:44]: The default value, so you can just remove it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Add a blue color for all links when link in hovering 
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
.

File: src/style.css (hunk 6 above)
Comment (vadym-valchak) [reply src/style.css:10]: I don't understand, what should I need to change here?

Comment (maxim2310) [reply src/style.css:10]: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties
image

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Almost there) Just one more thing needs to be fixed

**Review by lerastarynets — APPROVED** [review]
Keep up the great work!

**Pull request closed without merging**